### PR TITLE
User feedback running system commands

### PR DESCRIPTION
**Pull request by jk05**
Adjust the messaging shown to users with regards to system updates when running system commands

- Displays `1 system update,  no records` when `_systemUpdate` = 1
- Should display `n system update, no records`, when `_systemUpdate` > 1 (not been able to figure out to get this show in practice so suggestions welcome). I can remove this if its just deemed as unnecessary.
- Displays `no changes, no records` when !`_systemUpdate` (as before)

Ive only scoped the e2e tests within the `'serverVersion') >= 4.0`...

Screenshot 2019-12-10 at 11 34 06
Screenshot 2019-12-10 at 11 33 44

### CODE REVIEW

File: e2e_tests/integration/multi-db.spec.js [hunk 1]
```diff
@@ -38,6 +38,27 @@ describe('Multi database', () => {
...
+      cy.executeCommand(':clear')
+
+      cy.executeCommand(':use system')
+      cy.executeCommand('CREATE DATABASE test1')
+
+      cy.wait(3000) // CREATE database can take a sec
+
+      cy.resultContains('1 system update, no records')
+
+      cy.executeCommand('STOP DATABASE test1')
```
Comment (jk05) [inline e2e_tests/integration/multi-db.spec.js]: I know this isn't very pretty - doing `STOP DATABASE` twice in order to get 0 system updates. So suggestions welcome. 

I also haven't figured out a way to get >1 system updates to test pluralisation...

File: src/browser/modules/Stream/CypherFrame/helpers.js [hunk 2]
```diff
@@ -74,10 +72,15 @@ export function getBodyAndStatusBarMessages (result, maxRows) {
...
   }
 
+  const systemUpdatesValue = get(result, 'summary.counters._systemUpdates')
   const bodyMessage =
     (!updateMessages || updateMessages.length === 0) &&
     result.records.length === 0
-      ? '(no changes, no records)'
+      ? `(${(systemUpdatesValue > 0 &&
+          `${systemUpdatesValue} system update${(systemUpdatesValue > 1 &&
+            's') ||
```
Comment (jk05) [inline src/browser/modules/Stream/CypherFrame/helpers.js]: can remove pluralisation if its not deemed necessary

File: src/browser/modules/Stream/CypherFrame/helpers.js [hunk 3]
```diff
@@ -74,10 +72,15 @@ export function getBodyAndStatusBarMessages (result, maxRows) {
     streamMessage = streamMessage[0].toUpperCase() + streamMessage.slice(1)
   }
 
+  const systemUpdatesValue = get(result, 'summary.counters._systemUpdates')
```
Comment (jk05) [inline src/browser/modules/Stream/CypherFrame/helpers.js]: I have had to pull it here similar to another section in the code, no ideal but only way I can see to get this value (suggestions welcome).

I haven't defaulted to anything as if it's undefined thats fine, and the logic should still hold...

in terms of naming, `systemUpdatesValue` or `systemUpdatesCount`?

File: e2e_tests/integration/multi-db.spec.js (hunk 1 above)
Comment (jk05) [reply e2e_tests/integration/multi-db.spec.js]: Maybe I can add a unit test for the `/Stream/CypherFrame/helpers.js` instead for this. I will try this first and then return.

Comment (oskarhane) [reply e2e_tests/integration/multi-db.spec.js]: No, I think this is a good approach 👍

File: src/browser/modules/Stream/CypherFrame/helpers.js (hunk 3 above)
Comment (oskarhane) [reply src/browser/modules/Stream/CypherFrame/helpers.js]: I think this is the only place where it's available after serialization.

File: src/browser/modules/Stream/CypherFrame/helpers.js (hunk 2 above)
Comment (oskarhane) [reply src/browser/modules/Stream/CypherFrame/helpers.js]: 👍

**Pull request merged**